flink-tests/src/test/java/org/apache/flink/test/checkpointing/RescalingITCase.java

```diff
@@ -670,12 +676,15 @@ public void run(SourceContext<Integer> ctx) throws Exception {
 				}
 
 				Thread.sleep(100);
```

Comment:
I just kept the the original value from @tillrohrmann to minimize unrelated changes. I assume the concrete choice of 100ms is arbitrary so I am fine with the proposed change.